runtime/xamarin/runtime.h

```diff
@@ -149,6 +149,10 @@ enum NSObjectFlags {
 	NSObjectFlagsIsCustomType = 128,
 };
 
+enum NSObjectFlags2 {
```

Comment:
I suppose that if the new enum should be private/internal, this is not the right place for it.

Reply:
That's fine: https://github.com/xamarin/xamarin-macios/blob/0f12c101e0da3968fedb540ee75070f7f0ed5f82/runtime/xamarin/runtime.h#L4-L5